lib/matplotlib/widgets.py

```diff
@@ -1593,9 +1594,16 @@ def __init__(self, ax, labels, active=0, activecolor=None, *,
                 button.
 
             .. versionadded:: 3.7
+        layout_direction : {'vertical', 'horizontal'}
+            The orientation of the buttons: 'vertical' places buttons from top
+            to bottom, 'horizontal' places buttons from left to right.
```

Comment:
```suggestion
            to bottom, 'horizontal' places buttons from left to right, distributing
            the buttons across the whole Axes is divided into equal-sized boxes,
            and each button is left-aligned in the box. There is currently no size
            check with the associated labels, so that long labels may overlap with
            the next box.
```

Reply:
Sorry, I did not check what this actually does:

![grafik](https://github.com/user-attachments/assets/fb30ab7b-52cc-483e-a21b-034844d83ac1)

I strongly think we should follow standard conventions that labels are right of the button not above, because I consider the unusual layout a UX issue. See also https://github.com/matplotlib/matplotlib/pull/27946#issuecomment-2123384239.

![grafik](https://github.com/user-attachments/assets/a467ab6b-864e-412f-9cac-b7af893688b2)